src/Mod/Draft/draftguitools/gui_snaps.py

```diff
@@ -28,21 +28,81 @@
 # \brief Provide the Draft_Snap commands used by the snapping mechanism
 # in Draft.
 
-from PySide.QtCore import QT_TRANSLATE_NOOP
-
 import FreeCADGui as Gui
 import draftguitools.gui_base as gui_base
+
+from PySide import QtGui
+from PySide.QtCore import QT_TRANSLATE_NOOP
```

Comment:
Again these imports should occur first because they are not specific to FreeCAD.